changelog/3.9.0/kong/remove-datakit.yml

```diff
@@ -0,0 +1,2 @@
+message: "**Wasm**: Removed the experimental datakit Wasm filter"
```

Comment:
I don't think we need a changelog entry for this, after this PR it will never have been "in" (on a release)

Reply:
> after this PR it will never have been "in" (on a release)

datakit has been bundled with Kong since 3.7